ui/raidboss/data/06-ew/alliance/thaleia.txt

```diff
@@ -656,7 +658,7 @@ hideall "--sync--"
 5237.6 "Destructive Bolt" Ability { id: "8CEC", source: "Eulogia" }
 
 5300.0 label "Torrential Tridents"
-5297.8 "--middle--" Ability { id: "8A02", source: "Eulogia" }
+#5297.8 "--middle--" Ability { id: "8A02", source: "Eulogia" } # unreachable - info only
```

Comment:
I think we should keep these info only lines in timeline order for consistency.

Reply:
> I think we should keep these info only lines in timeline order for consistency.

That would require moving them above the labels, which serve as block headers, so I think that would hurt readability.  They also shouldn't ever sync, since the --middle-- happens before the 0x14 line happens, but the 0x14 is the jump to the label itself (see the entries at 4400.0).

I thought commenting them out to avoid sync order lint (as they're nonfunctional) was a reasonable compromise to keep them for context.  Do you think they should each go right above the label?

Reply:
We have other examples in the same file where a boss re-centering sync is right above a labeled block, so keeping them in-order would make the file self-consistent. [L63](https://github.com/OverlayPlugin/cactbot/blob/9fcd44834a3ebc179f47a7da53981272d5e5377a/ui/raidboss/data/06-ew/alliance/thaleia.txt#L63) [L198](https://github.com/OverlayPlugin/cactbot/blob/9fcd44834a3ebc179f47a7da53981272d5e5377a/ui/raidboss/data/06-ew/alliance/thaleia.txt#L198) [L605](https://github.com/OverlayPlugin/cactbot/blob/9fcd44834a3ebc179f47a7da53981272d5e5377a/ui/raidboss/data/06-ew/alliance/thaleia.txt#L605)
  We could also drop them, especially since I notice they don't exactly match logs that get that far into the fight as it is (some of the re-centerings don't happen where indicated, and others that do happen are missing).